### PR TITLE
Refactor InputToolbar component to FC

### PR DESCRIPTION
**Pull request by danielmejiadev**
- Refactor the input toolbar component using functional components and hooks.

@xcarpentier Could you please take a look?

### CODE REVIEW

File: src/InputToolbar.tsx [hunk 1]
```diff
@@ -47,128 +47,96 @@ export interface InputToolbarProps<TMessage extends IMessage> {
...
+    const keyboardWillShowListener = Keyboard.addListener('keyboardWillShow', () => setPosition('relative'));
+    return () => keyboardWillShowListener?.remove();
+  }, []);
+
+  useEffect(() => {
+    const keyboardWillHideListener = Keyboard.addListener('keyboardWillHide', () => setPosition('absolute'));
+    return () => keyboardWillHideListener?.remove();
+  }, []);
+
+  const renderActions = useCallbackOne(() => {
```
Comment (xcarpentier) [inline src/InputToolbar.tsx:63]: I don't really like a callback returning a component, can you move it directly under the return, please?

File: src/InputToolbar.tsx [hunk 2]
```diff
@@ -47,128 +47,96 @@ export interface InputToolbarProps<TMessage extends IMessage> {
...
-      return <Actions {...props} />
-    }
     return null
-  }
+  }, [props.renderActions, props.onPressActionButton]);
 
-  renderSend() {
-    if (this.props.renderSend) {
-      return this.props.renderSend(this.props)
+  const renderSend = useCallbackOne(() => {
```
Comment (xcarpentier) [inline src/InputToolbar.tsx:74]: The same message concerning callback returning a component, please move under return.

File: src/InputToolbar.tsx [hunk 3]
```diff
@@ -47,128 +47,96 @@ export interface InputToolbarProps<TMessage extends IMessage> {
...
-    return <Send {...this.props} />
-  }
 
-  renderComposer() {
-    if (this.props.renderComposer) {
-      return this.props.renderComposer(this.props as ComposerProps)
+    return <Send {...props} />
+  }, [props.renderSend]);
+
+  const renderComposer = useCallbackOne(() => {
```
Comment (xcarpentier) [inline src/InputToolbar.tsx:82]: The same message concerning callback returning a component, please move under return.

File: src/InputToolbar.tsx [hunk 4]
```diff
@@ -47,128 +47,96 @@ export interface InputToolbarProps<TMessage extends IMessage> {
...
     }
 
-    return <Composer {...this.props} />
-  }
+    return <Composer {...props} />
+  }, [props.renderComposer]);
 
-  renderAccessory() {
-    if (this.props.renderAccessory) {
+  const renderAccessory = useCallbackOne(() => {
```
Comment (xcarpentier) [inline src/InputToolbar.tsx:90]: The same message concerning callback returning a component, please move under return.

**Comment by danielmejiadev** [comment]
@xcarpentier Hi, I already solved your comments.

**Pull request merged**